src/search/components/dateTime.component.tsx

```diff
@@ -20,100 +19,95 @@ export interface UpdateFilterParams {
+      fromDateOrToDateChanged === 'toDate' &&
+      (!otherDate || !isBefore(date, otherDate));
+
+    if (validFromDate || validToDate) changeDate(date);
+  } else if (!date) {
+    changeDate(null);
   }
 }
 
-export interface DateTimeFilterProps {
-  onChange: (value: { fromDate?: string; toDate?: string }) => void;
-  value: { fromDate?: string; toDate?: string };
+export interface DateTimeSearchProps {
+  receivedFromDate: Date | null;
+  receivedToDate: Date | null;
+  changeFromDate: (fromDate: Date | null) => void;
+  changeToDate: (toDate: Date | null) => void;
 }
 
-export const DateTimeFilter = (
-  props: DateTimeFilterProps
-): React.ReactElement => {
-  const initialFromDate = props.value?.fromDate
-    ? new Date(props.value.fromDate)
-    : null;
-  const initialToDate = props.value?.toDate
-    ? new Date(props.value.toDate)
-    : null;
+const DateTimeSearch = (props: DateTimeSearchProps): React.ReactElement => {
+  const { receivedFromDate, receivedToDate, changeFromDate, changeToDate } =
+    props;
 
-  const [fromDate, setFromDate] = React.useState<Date | null>(initialFromDate);
-  const [toDate, setToDate] = React.useState<Date | null>(initialToDate);
+  const [fromDate, setFromDate] = React.useState<Date | null>(receivedFromDate);
+  const [toDate, setToDate] = React.useState<Date | null>(receivedToDate);
 
   const [popupOpen, setPopupOpen] = React.useState<boolean>(false);
 
   const invalidDateRange = fromDate && toDate && isBefore(toDate, fromDate);
 
   return (
     <Box
+      aria-label="date-time search box"
       sx={{
-        border: 'solid',
+        border: '1.5px solid',
+        borderColor: invalidDateRange ? 'rgb(214, 65, 65)' : undefined,
         borderRadius: '10px',
         display: 'flex',
         flexDirection: 'row',
       }}
     >
+      <CalendarMonth sx={{ fontSize: 40, padding: '10px 5px 0px 5px' }} />
       <LocalizationProvider dateAdapter={AdapterDateFns}>
-        <Grid container>
-          <Typography>From date/time</Typography>
+        <form style={{ display: 'inline-grid', padding: 3 }}>
+          <Typography>From date</Typography>
           <DateTimePicker
             inputFormat="yyyy-MM-dd HH:mm:ss"
             mask="____-__-__ __:__:__"
             value={fromDate}
             maxDateTime={toDate || new Date('2100-01-01 00:00:00')}
             componentsProps={{
-              actionBar: { actions: ['clear', 'cancel', 'accept'] },
+              actionBar: { actions: ['clear'] },
             }}
-            // onChange handles user-input date-time changes
             onChange={(date) => {
               setFromDate(date as Date);
-              if (datesEqual(date, fromDate)) return;
-              if ((date == null || isValid(date as Date)) && !popupOpen) {
+              if (!popupOpen) {
                 updateFilter({
                   date: date as Date,
-                  prevDate: initialFromDate,
+                  prevDate: fromDate,
```

Comment:
This reintroduces the bug we had previously, where "half" selecting a date doesn't work. We need to use `receivedFromDate`/`receivedToDate` in `prevDate`. See gif

![OG-timestamp-search-bug](https://user-images.githubusercontent.com/22525471/203573257-03165a4a-4320-4f1a-8369-0ad45e94a671.gif)

Reply:
Thanks for pointing that out. I'm annoyed I let that back in there. And the deadspace issue - a constant reminder I need to work on my CSS skills